src/Control/Map.h

```diff
@@ -1266,6 +1268,7 @@ class CMapView : public COleControl, IMapViewCallback
 	virtual void _FireValidateShape(LONG LayerHandle, IDispatch* Shape, tkMwBoolean* Cancel) { FireValidateShape(LayerHandle, Shape, Cancel); }
 	virtual void _FireAfterShapeEdit(tkUndoOperation NewShape, LONG LayerHandle, LONG ShapeIndex) { FireAfterShapeEdit(NewShape, LayerHandle, ShapeIndex); }
 	virtual void _FireShapeValidationFailed(LPCTSTR ErrorMessage) { FireShapeValidationFailed(ErrorMessage); }
+	virtual void _FireBeforeVertexDigitized(DOUBLE* pointX, DOUBLE* pointY) { FireBeforeVertexDigitized(pointX, pointY); }

```

Comment:
You've added these methods to allow the event to be called from the MapViewCallback.  Is that an intended/anticipated requirement?

Reply:
It was anticipated - for now I only call this event from Map_Events as that's the place where I can call it easily for all 'new' digitized points. But it would maybe make sense to also fire this (or another) event when the position of an existing vertex is changed, and then I thought we might need the callback?

Reply:
So I'll leave this in for now - unless you prefer to keep this callback to a minimum?

Reply:
The truth is that I don't know enough about the callback events to say one way or another.  So I'll agree, we can leave it for now.